tests/unit/test_credentials.py

```diff
@@ -3587,6 +3836,7 @@ def test_load_sso_credentials_without_cache(self):
             self.assertEqual(credentials.access_key, 'foo')
             self.assertEqual(credentials.secret_key, 'bar')
             self.assertEqual(credentials.token, 'baz')
+            self.assertEqual(credentials.account_id, '1234567890')
```

Comment:
Since account ID for SSO is attached directly to the provider, these existing tests will always have the value set. Because of this I think it makes more sense to just update the existing tests.